kasa/smartdimmer.py

```diff
@@ -41,19 +41,37 @@ def brightness(self) -> int:
         return int(sys_info["brightness"])
 
     @requires_update
-    async def set_brightness(self, value: int):
-        """Set the new dimmer brightness level in percentage."""
+    async def set_brightness(self, brightness: int, transition: int = None):
```

Comment:
I think it makes sense to make the transition a kwarg-only.

```suggestion
    async def set_brightness(self, brightness: int, *, transition: int = None):
```